src/components/home/footer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faFacebook,
+  faTwitter,
+  faGoogle,
+  faGithub,
+} from "@fortawesome/free-brands-svg-icons";
+import "./style.scss";
+
+const Footer = () => {
```

Comment:
footer changes in this pr?

can you isolate this PR only to the profile page files then send it again